services/workflows-service/src/filter/filter.controller.external.ts

```diff
@@ -59,11 +62,15 @@ export class FilterControllerExternal {
   @swagger.ApiForbiddenResponse()
   @UsePipes(new ZodValidationPipe(FilterCreateSchema, 'body'))
   async createFilter(@common.Body() data: FilterCreateDto) {
-    return await this.service.create({
+    const createdFilter = await this.service.create({
       data: {
         ...data,
         query: data?.query as InputJsonValue,
       },
     });
+    const websocketServerNotifyUri = `${env.WEBSOCKET_URL}/notify?type=filters`;
+    // todo is it important to await this?
+    await axios.post(websocketServerNotifyUri);
+    return createdFilter;
```

Comment:
@Omri-Levy what do you think?

Reply:
Removing the `await` does seem to make sense here. If you do, please confirm how a rejected promise behaves.